package.json

```diff
@@ -20,10 +20,16 @@
     "body-parser": "^1.18.3",
     "config": "^1.30.0",
     "cors": "^2.8.4",
+    "ejs": "^2.6.1",
```

Comment:
Ejs is a template engine why do you need it ? If you do not need it please remove.